identity-service/src/notifications/processNotifications/utils.js

```diff
@@ -0,0 +1,53 @@
+const BN = require('bn.js')
+
+const WEI = new BN('1000000000000000000')
+
+const formatNumberCommas = (num) => {
```

Comment:
Any reason to do this vs using something like https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat ?

EDIT: I see now that this is copied over, and everything seems fine, just curious since this seems like a lot of custom logic for something I _think_ we might be able to leverage existing tools for

Reply:
Yeah, this is better. Nice find, switching it over